test/new-e2e/tests/agent-platform/install-script/install_script_test.go

```diff
@@ -186,3 +209,38 @@ func (is *installScriptSuite) DogstatsdAgentTest() {
 	common.CheckInstallationInstallScript(is.T(), client)
 	is.testUninstall(client, "datadog-dogstatsd")
 }
+
+type installScriptSuiteSysVInit struct {
```

Comment:
**nit**: 
```suggestion
type installScriptSuiteDocker struct {
```

Reply:
The name of the suite reflects that the goal is to test on a system that uses SysV-init (since removing the SysV service files was what broke the installation). We use docker to test this because in practice that's the only place where this matters (given that systems old enough to default to sysv have too old of a glibc to run the Agent anyway), but it could've been implemented on a regular remote host with a recent enough OS and manually setting SysV as the init system instead of the default systemd (it just sounds so much harder and unlikely than the docker "use case").

What do you think?

Reply:
My main issue with the `SysVInit` is that we are can add more asserts on the install for the agent on a custom container that are not online limited to SysVInit. But I understand your point, if that is the case we we create other suites that use the docker client.